.travis.sh

```diff
@@ -0,0 +1,2 @@
+export SECRET_KEY='\x1b\xa5*G\xca\x88@\xe37\x8e\x8cP\x18\xef\xa3\xc0r\xaa\xf4\x94H3\xc3\xfd'
```

Comment:
Great job though I feel like this info should be stored in the env. I think there are also ways to configure env variables in travis